src/object-identifier.scss

```diff
@@ -0,0 +1,61 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-object-identifier;
+
+.#{$block} {
+  @mixin fd-object-identifier-title-base {
+    @include fd-reset();
+
+    color: var(--sapList_TextColor);
+  }
+
+  $fd-object-identifier-title-font-size: var(--sapFontLargeSize);
+  $fd-object-identifier-title-font-weight: bold;
+
+  &__title,
+  &__title--bold {
+    @include fd-object-identifier-title-base();
+  }
```

Comment:
That's not how modifier classes work.
Let's say you have a class `.fd-something__title` which applies some styling. Then you need to "extend" this styling by adding additional properties, or overwrite existing ones. Let's say you want to add bold to the text. You will need to do `fd-something__title fd-something__title--bold`. In this case `fd-something__title--bold` does not have a reset and inherits the text color and everything else from `fd-something__title`. `fd-something__title--bold` cannot be used without `fd-something__title`, meaning you can't have an element like this: `<p class="fd-something__title--bold">...</p>`.
It should be either `<p class="fd-something__title fd-something__title--bold">...</p>` OR `<p class="fd-something__title-bold">...</p>` where the second one is not really recommended.

Reply:
fixed